### PR TITLE
dash/climate-performance

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed, improved extremes change performance in climate demo.

### CODE REVIEW

**Review by goransle — APPROVED** [review]
LGTM

File: samples/dashboards/demo/climate/demo.js [hunk 1]
```diff
@@ -246,12 +246,17 @@ async function setupBoard() {
                                     activeTimeRange[1] !== e.max
                                 ) {
                                     activeTimeRange = [e.min, e.max];
-                                    await updateBoard(
+
+                                    console.time('updateBoard');
```
Comment (sebastianbochan) [inline samples/dashboards/demo/climate/demo.js:250]: Do we need it here?

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

**Review by pawellysy — APPROVED** [review]
LGTM!

**Review by bre1470 — APPROVED** [review]
When the `selectionTimeout` is removed, the change of extremes is still stuttering, so the core issue is not solved.

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**